source/developer/interactive-messages.rst

```diff
@@ -286,6 +287,18 @@ Context
       }
 
   Then, when the integration receives the request, it can act based on the action id.
+  
+  3. **Authenticating the server**.Sometimes, you might want to include a custome header in requests to be consumed by the integration. In such cases, you could add a Custom Header to your message, and it will be added to the HTTP Header field.
+ 
+    .. code-block:: text
+
+       {
+      "context": {
+        "action_id": "someunguessableactionid"
+        },
+       "custom_header": "some value",
+      }
+To setup custom request header, add `customer_header` parameter to your JSON message. Request headers that are sent this way,  will automatically have an additional header field with the key `X-Custom-Header` and value of what has been originally set in JSON message.  
```

Comment:
Propose adding two spaces at the beginning of line so it's intended with the rest of the section above it.

Also propose a few other minor tweaks to

```
  To set up a custom request header, add ``customer_header`` parameter to your JSON message. Request headers that are sent this way automatically have an additional header field with the key ``X-Custom-Header`` and a value originally set in the JSON payload.
```